### PR TITLE
[Applications][TCSACR-441] Add new APIs for resource control

### PR DESCRIPTION
**Pull request by jeremy-jang**
### API Changes ###

 - ACR: https://code.sec.samsung.net/jira/browse/TCSACR-441

Added:
 - IEnumerable<ResourceControl> ApplicationInfo.ResourceControls { get; }  // Property
 - string ResourceControl.ResType { get; }  // Property
 - string ResourceControl.MinResVersion { get; }  // Property
 - string ResourceControl.MaxResVersion { get; }  // Property
 - bool ResourceControl.IsAutoClose{ get; }  // Property

### CODE REVIEW

File: src/Tizen.Applications.Common/Tizen.Applications/ResControl.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+using System.Collections.Generic;
+using System.Text;
+
+namespace Tizen.Applications
+{
+    /// <summary>
+    /// Resource control.
+    /// </summary>
+    /// <since_tizen> 9 </since_tizen>
+    public class ResControl
```
Comment (myroot) [inline src/Tizen.Applications.Common/Tizen.Applications/ResControl.cs:27]: ❌ DO NOT use abbreviations or contractions as part of identifier names.
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/general-naming-conventions#using-abbreviations-and-acronyms

```suggestion
    public class ResourceControl
```

Comment (jeremy-jang) [reply src/Tizen.Applications.Common/Tizen.Applications/ResControl.cs]: I changed names to ResourceControl

**Review by myroot — APPROVED** [review]

File: src/Tizen.Applications.Common/Tizen.Applications/ResourceControl.cs [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Tizen.Applications
+{
+    /// <summary>
+    /// Resource control.
```
Comment (wiertel) [inline src/Tizen.Applications.Common/Tizen.Applications/ResourceControl.cs:24]: "control"? maybe more like information?

File: src/Tizen.Applications.Common/Tizen.Applications/ResourceControl.cs [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        /// The resource type.
+        /// </summary>
+        /// <since_tizen> 9 </since_tizen>
+        public string ResourceType { get; }
+
+        /// <summary>
+        /// The minimum version of required resource package.
+        /// </summary>
+        /// <since_tizen> 9 </since_tizen>
+        public string MinResourceVersion { get; }
```
Comment (wiertel) [inline src/Tizen.Applications.Common/Tizen.Applications/ResourceControl.cs:47]: can those properties be null to inform about missing information/unspecified, or rather an empty string?

Comment (jeremy-jang) [reply src/Tizen.Applications.Common/Tizen.Applications/ResourceControl.cs:47]: I set default value as null.

File: src/Tizen.Applications.Common/Tizen.Applications/ResourceControl.cs (hunk 2 above)
Comment (jeremy-jang) [reply src/Tizen.Applications.Common/Tizen.Applications/ResourceControl.cs]: I changed the summary.

**Pull request merged**